programs/openbook-v2/src/state/open_orders_account.rs

```diff
@@ -168,10 +168,13 @@ impl OpenOrdersAccount {
             market.fees_accrued += maker_fees as u128;
 
             if fill.maker_out() {
-                self.remove_order(fill.maker_slot as usize, fill.quantity);
+                self.remove_order(fill.maker_slot as usize, fill.quantity, fill.price);
```

Comment:
this doesn't seem correct, in oracle peg orders the amounts are locked at peg_limit. a simple test ensuring that these amounts are zero after filling an oracle peg bid will be quite useful